packages/assets-controllers/package.json

```diff
@@ -87,9 +87,9 @@
     "@metamask/network-controller": "^23.2.0",
     "@metamask/permission-controller": "^11.0.6",
     "@metamask/preferences-controller": "^17.0.0",
-    "@metamask/providers": "^18.1.1",
-    "@metamask/snaps-controllers": "^9.19.0",
-    "@metamask/snaps-sdk": "^6.17.1",
+    "@metamask/providers": "^21.0.0",
+    "@metamask/snaps-controllers": "^11.2.1",
```

Comment:
snaps-controllers bump to 11.2.1 is not specifically needed for this PR; i initially bumped to 11.2.0 to fix some ts compatibility issues then bumped to latest again.

I can update this if this is not wanted 🙏